aspnetcore/fundamentals/app-state/sample/src/WebAppSessionDotNetCore2.0App/Controllers/HomeController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http;
+using System;
+
+namespace WebAppSession.Controllers
+{
+    #region snippet1
+    public class HomeController : Controller
+    {
+        const string SessionKeyName = "_Name";
+        const string SessionKeyYearsMember = "_YearsMember";
+        const string SessionKeyDate = "_Date";
+
+        public IActionResult Index()
+        {
+            // Requires using Microsoft.AspNetCore.Http;
+            HttpContext.Session.SetString(SessionKeyName, "Rick");
+            HttpContext.Session.SetInt32(SessionKeyYearsMember, 3);
+            return RedirectToAction("SessionNameYears");
```

Comment:
In an effort to promote best practices, let's use the `nameof` operator here. The new line will look as follows:
```csharp
return RedirectToAction(nameof(SessionNameYears));
```

Reply:
fixed